org.sailfishos.usermanager.xml

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.sailfishos.usermanager">
+    <method name="users">
+        <arg direction="out" type="a(ss)" name="users"/>
+        <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QList&lt;SailfishUser&gt;"/>
+    </method>
+    <method name="addUser">
+        <arg direction="in" type="s" name="login_name"/>
+        <arg direction="in" type="s" name="user_name"/>
+    </method>
+    <signal name="userAdded">
+        <arg type ="s" name="login_name"/>
+        <arg type ="s" name="user_name"/>
+    </signal>
+    <method name="removeUser">
+        <arg direction="in" type="s" name="login_name"/>
```

Comment:
Is it better to use username or uid here? I can work with either though.

Reply:
Username a bit easier and already implemented.